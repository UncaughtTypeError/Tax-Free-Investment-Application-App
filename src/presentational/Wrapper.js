import React from 'react';
// Components
import WrapperErrorBoundary from '../error-boundaries/WrapperErrorBoundary';
import Form from '../containers/Form';
// Theme
import Grid from '@material-ui/core/Grid';

const Wrapper = () => {

    return (
        <WrapperErrorBoundary>
            <Grid container alignItems="center" justify="center">
                <Grid item xs={10} sm={8} md={8} lg={8} xl={8}>
                    <Form />
                </Grid>
            </Grid>
        </WrapperErrorBoundary>
    );
}

export default Wrapper;