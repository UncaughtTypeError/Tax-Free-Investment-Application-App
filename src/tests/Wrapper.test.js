import React from 'react';
// Enzyme
import { shallow } from 'enzyme';
// Component
import Wrapper from '../presentational/Wrapper';

it('should render component with grid layout structure', () => {
    expect(shallow(<Wrapper />)).toMatchSnapshot();
});