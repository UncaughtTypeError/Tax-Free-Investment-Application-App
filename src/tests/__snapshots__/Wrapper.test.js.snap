// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component with grid layout structure 1`] = `ShallowWrapper {}`;
